crates/iracing-data-api-client/docs/ResultsApi.md

```
@@ -0,0 +1,243 @@
+## get_results_season_results
+
+> models::IracingApiResponse get_results_season_results(season_id, event_type, race_week_num)
+
+
+### Parameters
+
+
+Name | Type | Description  | Required | Notes
+------------- | ------------- | ------------- | ------------- | -------------
+**season_id** | **f64** |  | [required] |
+**event_type** | Option<[**IracingEventType**](.md)> | Retrict to one event type: 2 - Practice; 3 - Qualify; 4 - Time Trial; 5 - Race |  |
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)